simnode/dbtypes/scheme.py

```diff
@@ -51,7 +51,6 @@ class InitialState:
 class Obstacle(ABC):
     x: float
     y: float
-    height: float
     # oid: Optional[str] = None  # NOTE Limitation of inheritance from dataclasses
     # x_rot: Optional[float] = 0  # NOTE Limitation of inheritance from dataclasses
     # y_rot: Optional[float] = 0  # NOTE Limitation of inheritance from dataclasses
```

Comment:
For sake of striving for perfection you could add another comment similar to these for "z" such that it makes clear that every obstacle is supposed to have a z value.